commerce-configuration-1/ext.config.yaml

```diff
@@ -0,0 +1,99 @@
+operations:
```

Comment:
can you elaborate on the general concept about this and how it is used in config management? 
first time I see

Reply:
found this looking in the docs but not sure about its effect https://developer.adobe.com/app-builder/docs/guides/app_builder_guides/configuration/configuration#definition-of-dxasset-computeworker1